Docs + Examples + Tests (#5)
arthurbr11/sentence-transformers: docs/package_reference/sparse_encoder/models.md

> # Models

Reviewer: ```suggestion
# Modules
```
Same as above

`sentence_transformers.sparse_encoder.models` defines different building blocks, that can be used to create SparseEncoder networks from scratch. For more details, see [Training Overview](../../sparse_encoder/training_overview.md).

> Note that models from `sentence_transformers.models` can be used to such as `sentence_transformers.models.Transformer` see [SentenceTransformer Models](../sentence_transformer/models.md)

Reviewer: ```suggestion
Note that modules from `sentence_transformers.models` can also be used for Sparse models, such as `sentence_transformers.models.Transformer` from [SentenceTransformer > Modules](../sentence_transformer/models.md)
```
